ports/vhr/README.md

```diff
@@ -0,0 +1,8 @@
+## Installation
+Purchase the game on Steam and copy all the data to `ports/vhr/gamedata`. On first run the game will be patched.
```

Comment:
Instructions are only in port.json. 

Readme should include thank you notes with URL to the gamepage controls.

Example: https://github.com/PortsMaster/PortMaster-New/blob/main/ports/tvruhh/README.md
https://portmaster.games/packaging.html#readme-md

Reply:
Literally all my other readmes follow the same format but ok.